Feat: Add debugging logs control (#467)
denodrivers/postgres: README.md

| 1.17.0        | 0.15.0             | 0.17.1             |                      |
| 1.40.0        | 0.17.2             |                    | Now available on JSR |

## Breaking changes

> Although the `deno-driver` is pretty stable and robust, it is a WIP and we're

Reviewer: ```suggestion
Although `deno-postgresql` is pretty stable and robust, it is a WIP, and we're
```

still exploring the design. Expect some breaking changes as we reach version 1.0

> and enhance the feature set. PLease check the Releases for more info on breaking

Reviewer: ```suggestion
and enhance the feature set. Please check the Releases for more info on breaking
```
It'd be good to have the `Releases` be a link.

changes. Please reach out if there are any undocumented breaking changes.

## Found issues?

> Please file an issue with any problems with the driver in this repository's

Reviewer: ```suggestion
Please [file an issue](https://github.com/denodrivers/postgres/issues/new/choose) with any problems with the driver in this repository's
```

> issue section. If you would like to help, please look at the issues as well. You

Reviewer: ```suggestion
issue section. If you would like to help, please look at the [issues](https://github.com/denodrivers/postgres/issues) as well. You
```
